src/browser-lib/single-spa-angular.ts

```diff
@@ -39,111 +41,137 @@ export default function singleSpaAngular(userOpts: SingleSpaAngularOpts): LifeCy
     bootstrap: bootstrap.bind(null, opts),
     mount: mount.bind(null, opts),
     unmount: unmount.bind(null, opts),
-    update: opts.updateFunction
+    update: opts.updateFunction,
   };
 }
 
-function bootstrap(opts, props) {
-  return Promise.resolve().then(() => {
-    // In order for multiple Angular apps to work concurrently on a page, they each need a unique identifier.
-    opts.zoneIdentifier = `single-spa-angular:${props.name || props.appName}`;
-
-    // This is a hack, since NgZone doesn't allow you to configure the property that identifies your zone.
-    // See https://github.com/PlaceMe-SAS/single-spa-angular-cli/issues/33,
-    // https://github.com/single-spa/single-spa-angular/issues/47,
-    // https://github.com/angular/angular/blob/a14dc2d7a4821a19f20a9547053a5734798f541e/packages/core/src/zone/ng_zone.ts#L144,
-    // and https://github.com/angular/angular/blob/a14dc2d7a4821a19f20a9547053a5734798f541e/packages/core/src/zone/ng_zone.ts#L257
-    opts.NgZone.isInAngularZone = function() {
-      // @ts-ignore
-      return window.Zone.current._properties[opts.zoneIdentifier] === true;
-    }
+async function bootstrap(opts: BootstrappedSingleSpaAngularOpts, props: any): Promise<void> {
+  await Promise.resolve();
```

Comment:
Seems like a lot of changes, but it's only changed from `promise.resolve().then()` to `await Promise.resolve()`.

Reply:
This line is unnecessary - lifecycle functions such as bootstrap just need to return a promise, which is now being accomplished by making them async functions (which always return promises)

```suggestion
```